cmd/virt-launcher/virt-launcher.go

```diff
@@ -329,6 +330,13 @@ func main() {
 
 	log.InitializeLogging("virt-launcher")
 
+	// check if virt-launcher verbosity should be changed
+	if verbosityStr, ok := os.LookupEnv("VIRT_LAUNCHER_LOG_VERBOSITY"); ok {
+		verbosity, _ := strconv.Atoi(verbosityStr)
```

Comment:
i don't know how this VIRT_LAUNCHER_LOG_VERBOSITY env var could ever not be a valid integer, but i have no idea what happens if this error is ignored. do you think we should handle this error?

Reply:
done